src/main/java/uk/gov/hmcts/ccd/domain/service/search/global/GlobalSearchResponseTransformer.java

```diff
@@ -59,9 +59,8 @@ public GlobalSearchResponseTransformer(@Qualifier(CachedCaseDefinitionRepository
 
     public GlobalSearchResponsePayload.ResultInfo transformResultInfo(final Integer maxReturnRecordCount,
                                                                       final Integer startRecordNumber,
-                                                                      final Long totalSearchHits,
                                                                       final Integer recordsReturnedCount) {
-        final boolean moreToGo = totalSearchHits > (maxReturnRecordCount + startRecordNumber - 1);
+        final boolean moreToGo = recordsReturnedCount > (maxReturnRecordCount + startRecordNumber - 1);
 
         return GlobalSearchResponsePayload.ResultInfo.builder()
             .casesReturned(recordsReturnedCount)
```

Comment:
I think I understand the intended logic here, but original naming of parameters are misleading in terms of their immediate meanings as we read them. So, perhaps it could be good to rename them to better align with their semantics.